environments/production/app-processes.yml

```diff
@@ -72,6 +72,8 @@ celery_processes:
       prefetch_multiplier: 1
     ush_background_tasks:
       concurrency: 8
+    soltech_custom_script_queue:  # TODO: Remove after executing script
+      concurrency: 4
```

Comment:
An open question I have is whether a concurrency of 4 is enough? This totals to 16 processes (4 * 4 machines) which should definitely speed up the runtime, but I'm wondering if this should be taken up any higher?

Reply:
I agree, the estimate was more so just a very basic linear estimation. I believe things like rate limiting and pillow lag would greatly affect the actual speed at which the Celery processes can get through the cases.

Just an FYI that I've closed off this PR since we'll be running the script without Celery.

Reply:
okay,

I believe rate limiting won't apply here since we are submitting system forms directly.